ui/raidboss/data/05-shb/trial/varis-ex.js

```diff
@@ -359,10 +360,9 @@
         'Ventus Est': 'Ventus Est',
       },
       'replaceText': {
+        '\\?': '?',
```

Comment:
Sorry, can you explain what this line is doing?

Reply:
Hi sure. 

This lines are in text file:
`519.7 "--clones appear--" sync /:Phantom Varis:4CB3:/`
This line is listed in the `common.js` file

`294.0 "--clones appear?--" #sync /:Phantom Varis:4CB3:/`
this line has`?` more. If I delete this line for example in de: `'\\?': '?',`, that gives an error.
![varys-ex](https://user-images.githubusercontent.com/61565834/84988173-d0113280-b141-11ea-8920-1110c027ad4f.PNG)

Having it corrects this error. I don't know of any other solution.

Reply:
I think this text `--clones appear?--` here is just a fake syncing text for placeholding, maybe the `?` can be deleted?


And a little suggestion here:
when you try to share your cmd output information, you can just copy those text to comment, and quote them with \`\`\` and \`\`\`.

just like this:

>\`\`\`
>// your output info
>\`\`\`

then it comes to be:

```
// your output info
```

Reply:
The problem is that this translation does not do anything.

If the original text is `--clones appear?--` then `'\\?': '?'` translates `?` into `?` leaving you with the original unchanged `--clones appear?--`.  Therefore, that line does not translate anything, it only suppresses the error from testing.  That is my concern.

I see two options:
 1) translate `--clones appear\\?--` as its own line
 2) Change the common translation like this:

```
  '--clones appear(\\?)?--': {
    de: '--klone erscheinen$1--',
    fr: '--apparition des clones$1--',
    ja: '--幻影出現$1--',
    cn: '--克隆 体 出现$1--',
    ko: '--분신 소환$1--',
  },
```

Unfortunately, I don't think there's any way to add a space before the optional question mark here for French.  So if you want it to say `--apparaition des clones ?--` then option 1 is the only approach that would work.

Similarly, I think for the targetable one in #1484 you could do somethinglike:

```
  '--([0-9]+x )?targetable--': {
    de: '--$1anvisierbar--',
    fr: '--$1ciblable--',
    ja: '--$1ターゲット可能--',
    cn: '--$1可选中--',
    ko: '--$1타겟 가능--',
  },
```

And this would allow `--targetable--` to translate as `--ciblable--` and `--2x targetable--` to be `--2x ciblable--` with the same line.